src/main/resources/application.properties

```diff
@@ -117,3 +117,6 @@ workflow.service.host=localhost:${server.port}
 spring.aop.proxy-target-class=true
 
 logging.level.net.boomerangplatform.service.refactor=DEBUG
+
+workflow.mongo.collection.prefix=cicd
```

Comment:
should this default to flow, and then we adjust to CICD?

Reply:
I did intend this to be blank instead of cicd.